src/components/NavigationBar/navBar.scss

```diff
@@ -1,13 +1,14 @@
-@import "~@edx/brand/paragon/fonts.scss";
-@import "~@edx/brand/paragon/variables.scss";
-@import "~@edx/paragon/scss/core/core.scss";
-@import "~@edx/brand/paragon/overrides.scss";
+@import '~@edx/brand/paragon/fonts.scss';
+@import '~@edx/brand/paragon/variables.scss';
+@import '~@edx/paragon/scss/core/core.scss';
+@import '~@edx/brand/paragon/overrides.scss';
 
-$fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome";
+$fa-font-path: '~font-awesome/fonts';
+@import '~font-awesome/scss/font-awesome';
```

Comment:
Why make changes that are not necessary? This will make transition and blaming hard for later.

if they're using a linting config, kindly set your editor so follow those rules.